manifests/init.pp

```diff
@@ -329,19 +330,18 @@
   $snmptrapd_config             = $snmp::params::snmptrapd_config,
   Boolean $manage_client        = $snmp::params::manage_client,
   $snmp_config                  = $snmp::params::snmp_config,
-  $ensure                       = $snmp::params::ensure,
   Boolean $autoupgrade          = $snmp::params::autoupgrade,
   $package_name                 = $snmp::params::package_name,
   $snmpd_options                = $snmp::params::snmpd_options,
   $service_config_perms         = $snmp::params::service_config_perms,
   $service_config_dir_group     = $snmp::params::service_config_dir_group,
-  $service_ensure               = $snmp::params::service_ensure,
+  Enum['running', 'stopped'] $service_ensure = $snmp::params::service_ensure,
```

Comment:
Should we use https://github.com/puppetlabs/puppetlabs-stdlib#stdlibensureservice